README.md

```diff
@@ -30,10 +31,12 @@ Package contains several configuration presets:
  - Evonet - masternode with Evonet configuration
  - Testnet - masternode with testnet configuration
 
-There are two ways to apply a present:
+There are two ways to apply a preset:
  1. Rename corresponding dotenv file (i.e. `.env.local`) to `.env`
```

Comment:
I would move to a separate section like how to use `docker-compose`.

Reply:
Actually should we be using `docker-compose` at all anymore? If you start a masternode with `mn start [args]` it will ignore any settings (like the BLS privkey) made in the `.env` file, so should we remove documentation on starting with `docker-compose` completely? Or should I document in separate sections how to do configuration and startup with both the CLI and docker-compose + .env files?

Reply:
Yeah, I think we need to add a separate section at the bottom how to use docker-compose directly. It could be useful if you need to check logs and so on.

Reply:
Talking about sections:

We follow usally https://github.com/RichardLitt/standard-readme

Docker, downloading repo and npm install should go to Install section.

Presets explanation, start, stop, reset, masternode registration, using via docker-compose should go to Usage

Reply:
Cool, I had no idea about standard readme until now. I have applied the specification, and documented the CLI and Docker Compose steps separately, let me know what you think.